lib/supabase/mock.ts

```
@@ -0,0 +1,358 @@
+              }
+            })
+          }, 100)
+
+          return { data: newItem, error: null }
+        }
+      })
+    }),
+
+    update: (values: any) => ({
+      eq: (column: string, value: any) => ({
+        then: async (callback: any) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)